Add new explore folder (#10211)
getsentry/sentry-docs: docs/product/explore/index.mdx

---
title: Explore
sidebar_order: 40
description: Learn about the different ways you can explore your data in Sentry.
---

> The Explore tab in Sentry offers multiple ways to search and investigate your application's data. Look through your app's traces, metrics, profiles, and replays to get more information to help you debug issues.

Reviewer: ```suggestion
The Explore tab in Sentry offers multiple ways to search and investigate your application's data. You can look at your app's traces, metrics, profiles, and replays to get the information you need to debug issues more quickly and easily.
```
